### PR TITLE
FunctionCallParametersCheck: Add native parameter type

### PR DESCRIPTION
**Pull request by staabm**
just stumbled over this untyped parameter

### CODE REVIEW

File: src/Rules/FunctionCallParametersCheck.php [hunk 1]
```diff
@@ -61,7 +60,7 @@ public function check(
 		ParametersAcceptor $parametersAcceptor,
 		Scope $scope,
 		bool $isBuiltin,
-		$funcCall,
+		Expr\CallLike $funcCall,
```
Comment (ondrejmirtes) [inline src/Rules/FunctionCallParametersCheck.php]: Please use exactly the union type in the PHPDoc. CallLike is wider than that - there are more classes there that are not supposed to be passed into FunctionCallParametersCheck.

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]

**Comment by ondrejmirtes** [comment]
I rebased on 2.0.x (no reason to do this on 1.12.x, I'd be a bit afraid). But there's a conflict now :)

**Comment by staabm** [comment]
rebased

**Comment by ondrejmirtes** [comment]
Thank you!

**Pull request merged**